README.md

```diff
@@ -162,6 +163,9 @@ cukinia_listen4 udp 53
 # Check the link interfaces point to /tmp/interfaces
 cukinia_symlink /etc/network/interfaces /tmp/interfaces
 
+# Retry the test 3 times with 1 second interval before failing
+retry 3 1 cukinia_test -f /etc/hostname
```

Comment:
IMO the `retry 3 1` syntax looks kind of cryptic which goes against cukinia's intent to be kind of "english understandable" like cucumber. Any way to make it less so ?

Something like this would be neat. A "on" element would allow to condition the retry to a fail (may be default) or pass as this could be useful too.

``` bash
as "The test description" \
  when "image_is_production" \
    on failure retry 3 after 3s \
      cukinia_cmd /some/testname
```

Reply:
I tried a first "on" implementation. Needs maybe a second opinion here